Signal/src/ViewControllers/NewGroupViewController.m

```diff
@@ -97,8 +99,6 @@ - (void)loadView
     [super loadView];
 
     self.title = NSLocalizedString(@"NEW_GROUP_DEFAULT_TITLE", @"The navbar title for the 'new group' view.");
-    self.navigationItem.leftBarButtonItem =
-        [self createOWSBackButtonWithTarget:self selector:@selector(backButtonPressed:)];
```

Comment:
I was mostly able to remove the custom back buttons - except in conversation view where we're deliberately changing its appearance.